src/applications/mhv/secure-messaging/components/Navigation.jsx

```diff
@@ -1,92 +1,29 @@
-import React, { useEffect, useState } from 'react';
-
-const Navigation = () => {
-  const [isMobile, setIsMobile] = useState(false);
-  const [isNavigationOpen, setIsNavigationOpen] = useState(false);
-  function openNavigation() {
-    setIsNavigationOpen(true);
-  }
-
-  function closeNavigation() {
-    setIsNavigationOpen(false);
-  }
-
-  function checkScreenSize() {
-    if (window.innerWidth <= 481 && setIsMobile !== false) {
-      setIsMobile(true);
-    } else {
-      setIsMobile(false);
-      setIsNavigationOpen(false);
-    }
-  }
-
-  function openNavigationBurgerButton() {
-    return isMobile ? (
-      <div className="va-btn-sidebarnav-trigger">
-        <div className="button-background" />
-        <div className="button-wrapper">
+import React from 'react';
+
+const Navigation = () => (
+  <div className="side-menu-nav">
+    <ul>
+      <li>
+        <div className="nav-title">
+          <i className="medkit-icon fas fa-medkit" />

```

Comment:
Since our team has been working in a separate dedicated branch, we are now submitting individual pull requests with outstanding work from all our team members. This component is modified and will be updated in one of the following PRs shortly. Please disregard this